operator/src/main/java/oracle/kubernetes/operator/Watcher.java

```diff
@@ -93,6 +93,11 @@ void waitForExit() {
     }
   }
 
+  // for test
```

Comment:
Sorry, I hit enter by mistake in my previous comment. I was thinking that getNewResourceVersion() method of `Watcher.java` will check if event type is `BOOKMARK` and then update the resource version sent as part of bookmark event. Please let me know if my understanding is incorrect or if I missed something.

Reply:
If you follow the logic through, you will see that this is what happens. I'm working on another branch, but tomorrow I'll find the existing code.

Reply:
ok, got it. I wonder if we need to test it functionally (in integration test or performance test) to verify if bookmark events helps in mitigating the impact of short event history window. In any case, this looks fine to me.